database_map_server/BaseMap map
---
bool response
string errorMessage